bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -73,10 +72,10 @@ export class UIEvents
     else if et == "move"
       logger.debug("Registering tool: #{type} for event '#{et}'")
       if tool_view._move_enter?
-        tool_view.listenTo(@, "move:enter", tool_view._move_enter)
-      tool_view.listenTo(@, "move", tool_view["_move"])
+        tool_view.listenTo(@, "move:enter:#{id}", tool_view._move_enter)
```

Comment:
Yes, that is my understanding. I'm suggesting that anything that still responds to `"move:enter"` will still work with this change. But perhaps that's incorrect (in which case) everything that listened to `"move:enter"` has to be updated. (and maybe has been, I have not checked)